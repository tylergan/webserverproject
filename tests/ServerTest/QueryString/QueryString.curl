curl 127.0.0.1:8070/cgibin/QUERY_STRING.py?i=20&t=23