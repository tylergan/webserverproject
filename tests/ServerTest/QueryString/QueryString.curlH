curl -I 127.0.0.1:8070/cgibin/QUERY_STING.py?i=20&t=23