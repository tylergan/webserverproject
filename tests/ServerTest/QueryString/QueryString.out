i=20&t=23
